PhotoRoom/react-template
# Feat: add foldering

This PR implements these user stories:
- As a user, I can see the list of images previously sent to the server from my computer. They all appear in an "Untitled Folder" folder by default.

## How it was implemented
- `useImageLibrary` takes care of interacting with library state
- `<LibrarySideBar />` displays the a tree view of folder and image and allows navigation
- `<ImagePreview />` displays a preview of an image and of its result without background

## Test instructions
- upload in a image 
- => it should appear in "Untitled Folder"
- click on "Without Background"
- => you should see the image without background
- upload an other image 
- => it should appear in "Untitled Folder"
- go back to previous image by clicking its name
